modules/monitoring-creating-cluster-id-labels-for-metrics.adoc

```diff
@@ -157,10 +157,4 @@ data:
 <2> Specify the name of the cluster ID label for metrics sent to remote write storage. If you use a label name that already exists for a metric, that value is overwritten with the name of this cluster ID label. For the label name, do not use `+++__tmp_openshift_cluster_id__+++`. The final relabeling step removes labels that use this name.
 <3> The `replace` write relabel action replaces the temporary label with the target label for outgoing metrics. This action is the default and is applied if no action is specified.
 
-. Save the file to apply the changes to the `ConfigMap` object.
-The pods affected by the updated configuration automatically restart.
-+
-[WARNING]
-====
-Saving changes to a monitoring `ConfigMap` object might redeploy the pods and other resources in the related project. Saving changes might also restart the running monitoring processes in that project.
-====
+. Save the file to apply the changes. The new configuration is applied automatically.
```

Comment:
@juzhao  When I tested this, this change did not trigger pods to redeploy so I am not including the sentence about the redeployment here.

Reply:
yes, mentioned pods won't restart in https://issues.redhat.com/browse/OBSDOCS-1155